### PR TITLE
[WNMGDS-1121] icon css refs cleanup

### PR DESCRIPTION
**Pull request by scheul93**
## Summary
Updated components to use inline SVGs instead of svgs as css background images

_Please note: USA Banner will be in a separate PR because it required some refactor & talking to Patrick, we will not be updating the `Dropdown` component because it's more of a look and feel than an icon_

### Removed
Removing references to .svg files in css of the following components
* Alert
* Accordion
* Vertical Nav
* Step List

## How to test
Run doc site locally and confirm that changed components look like their original versions

See demo sites at:
* Storybook - http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1121/icon-css-refs-cleanup/core/storybook/?path=/story/components-accordion--controlled
* Doc site - http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1121/icon-css-refs-cleanup/core/

### CODE REVIEW

File: packages/design-system/src/components/StepList/Step.jsx [hunk 1]
```diff
@@ -75,7 +76,12 @@ export const Step = ({ step, ...props }) => {
         )}
       </div>
       <div className="ds-c-step__actions" {...actionsLabel}>
-        {step.completed && <div className="ds-c-step__completed-text">{props.completedText}</div>}
+        {step.completed && (
+          <div className="ds-c-step__completed-text">
+            <CheckIcon className="ds-c-icon-color--success" />
```
Comment (pwolfert) [inline packages/design-system/src/components/StepList/Step.jsx:81]: There's a difference in placement:

steplist

File: packages/design-system/src/components/Alert/Alert.tsx [hunk 2]
```diff
@@ -195,6 +216,7 @@ export class Alert extends React.PureComponent<
         aria-labelledby={heading ? this.headingId : undefined}
         {...alertProps}
       >
+        {this.getIcon()}
```
Comment (pwolfert) [inline packages/design-system/src/components/Alert/Alert.tsx:219]: Alert size changed slightly but not position. Seems fine to me.

alerts

File: packages/design-system/src/components/VerticalNav/VerticalNavItemLabel.jsx [hunk 3]
```diff
@@ -64,7 +65,12 @@ export class VerticalNavItemLabel extends React.PureComponent {
       props = Object.assign(props, this.anchorProps());
     }
 
-    return <this.LabelComponent {...props}>{this.props.label}</this.LabelComponent>;
+    return (
+      <this.LabelComponent {...props}>
+        {this.props.label}
+        {this.props.hasSubnav && <ArrowIcon direction={this.props.collapsed ? 'down' : 'up'} />}
```
Comment (pwolfert) [inline packages/design-system/src/components/VerticalNav/VerticalNavItemLabel.jsx:71]: For this one, the weight feels different

vertical-nav

Comment (scheul93) [reply packages/design-system/src/components/VerticalNav/VerticalNavItemLabel.jsx:71]: The weight here is different. As part of consolidating all the arrow icons in the design system, we had to go with a lighter-weight icon.

**Review by pwolfert — APPROVED** [review]
LGTM 👍

**Review by forrestbaer — APPROVED** [review]
looks good!

**Pull request merged**